OpenUpSA/wazimap-ng-ui
# Hotfix null metadata source

## Description

Fix null values in metadata crashing the chart image download preparation code.

## Related Issue

https://wazimap.atlassian.net/browse/WNCM-237?atlOrigin=eyJpIjoiOGQ1N2Y3N2FlMTcwNGMxNmE4Mzc0N2Y2NTZmYWJiMjUiLCJwIjoiaiJ9

## How is it tested automatically?

Not yet - hotfix

## How should a reviewer test it locally

visit http://localhost:1234/#geo:MP313 and observe the page loading without errors

## Screenshots

![Screenshot_2022-01-27_16-58-47](https://user-images.githubusercontent.com/235801/151384807-35da3735-4abe-44d8-841f-0895a69ecb17.png)

![Screenshot_2022-01-27_17-02-16](https://user-images.githubusercontent.com/235801/151384946-eb54877d-9aad-4bf5-b370-f0d2dba2e95b.png)


## Changelog

### Added

condition to exclude metadata.source from image metadata if it's null (like if it's undefined)

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
